app/controller/index.rb

```diff
@@ -0,0 +1,10 @@
+class Chitter < Sinatra::Base
+
+  get '/' do
+    flash[:notice] = nil
+    flash[:errors] = nil
+    @cheets = Cheet.all(:order =>[:time.desc])
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator '=>'.